### ISSUE TITLE
folder drag & drop

### ISSUE THREAD

**Issue by hyperfx** [darbyjohnston/DJV#345]
Currently to view a frame sequence, we have to open a folder and drag & drop the first frame of a sequence. Would it be possible to include an option to be able to drab & drop the folder icon instead of a frame inside the folder?

**Comment by darbyjohnston** [comment]
Hi, that's a good suggestion; I guess if you have multiple sequences in the folder it would also open those? It could also potentially be recursive if you have sub-directories with sequences, though that might be dangerous if you accidentally drag and dropped a folder with a deep hierarchy (like your home or root folder).

**Comment by hyperfx** [comment]
Maybe if there's was more than one sequence, or sub folders of any kind, it presented a popup window for you to verify which you'd like to open? Hmmm, that's probably not the best solution.

**Comment by darbyjohnston** [comment]
The dialog is not a bad idea but that could also get fairly heavy if you drag and dropped a folder with a deep hierarchy (like would it display a tree view of the directory structure?). Maybe a good start would be to just have it open all the files in the folder (non-recursively), and later on consider adding a dialog if more fine-grained control is needed.

**Comment by hyperfx** [comment]
Yeah, sounds like a good first step. As a VFX artist, I keep all my frame sequences pretty organized in folders, so that's the only reason I think I'd prefer it that way.
![sequence_folders](https://user-images.githubusercontent.com/5239056/91100770-9ca7c400-e61a-11ea-9402-a1978739a30b.JPG)

**Comment by darbyjohnston** [comment]
Hi, after a long delay I have released a new version, 3.0.0, which has support for folder drag and drop. It will also open multiple sequences within the folder. I'm going to close this issue as done, but feel free to open a new issue if there any problems with the new version.
